incident/mysql/incident.go

```diff
@@ -43,7 +44,9 @@ func NewMySQLReporter(ctx context.Context, db *sql.DB, source string) (incident.
 
 // Log records an incident with the given details.
 func (m *mysqlReporter) Log(ctx context.Context, baseURL, summary, category, fullURL, details string) {
-	if _, err := m.stmt.ExecContext(ctx, m.source, baseURL, summary, category, fullURL, details); err != nil {
+	now := time.Now()
+	glog.Errorf("%s: %s (%s %s)\n  %s at %v", baseURL, summary, category, fullURL, details, now)
```

Comment:
nit: You're assuming that `details` is written in such a way that "at <time>" can be suffixed in a non-confusing way. Perhaps use the time as a prefix instead?

Reply:
Done.